### PR TITLE
Add new join optimizer property to documentation

### PR DESCRIPTION
**Pull request by abhinavmuk04**
## Description
Add new join optimizer property to documentation

## Motivation and Context
Documentation update for treat-low-confidence-zero-estimation-as-unknown property

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

Pull branch, local docs build, everything looks good. Thanks!

**Comment by steveburnett** [comment]
@abhinavmuk04, is this ready to be merged? If so, remove the Draft status when you are ready.

**Comment by abhinavmuk04** [comment]
> @abhinavmuk04, is this ready to be merged? If so, remove the Draft status when you are ready.

@steveburnett Just waiting for the change to get one final approval then I'll merge! Thanks

**Pull request merged**
